frontend/src/styles/journal.css

```diff
@@ -43,4 +43,19 @@
   display: block;
   font-size: 14px;
   
-}
\ No newline at end of file
+}
+
+
+/** log in page related css */
+.google-logo {
+  width: 25px;
+  height: auto;
+}
+
+.register-link {
+  color:forestgreen;
```

Comment:
I might change this to the specific hex code just to keep it consistent with the other page.